infra-as-code/bicep/orchestration/hubPeeredSpoke/hubPeeredSpoke.bicep

```diff
@@ -0,0 +1,206 @@
+/*
+SUMMARY: This module acts as an orchestration module that create and configures a spoke network to deliver the Azure Landing Zone Hub & Spoke architecture
+DESCRIPTION:
+
+AUTHOR/S: KiZach
+VERSION: 1.0.0
+*/
+
+// **Parameters**
+// Generic Parameters - Used in multiple modules
+@description('The region to deploy all resoruces into. DEFAULTS TO = northeurope')
+param parLocation string = 'westeurope'
+
+@description('Prefix for the management group hierarchy. DEFAULTS TO = alz')
+@minLength(2)
+@maxLength(10)
+param parTopLevelManagementGroupPrefix string = 'alz'
+
+@description('Subscription Id to the Virtual Network Hub object. DEFAULTS TO empty')
+param parPeeredVnetSubscriptionId string = ''
+
+@description('Array of Tags to be applied to all resources in module. Default: empty array')
+param parTags object = {}
+
+@description('Set Parameter to true to Opt-out of deployment telemetry DEFAULTS TO = false')
+param parTelemetryOptOut bool = true
+
+// Subscription Module Parameters
+@description('The Management Group Id to place the subscription in. DEFAULTS TO empty')
+@allowed([
```

Comment:
Lets remove this list of '@allowed' inputs as whilst I like its intent, we should keep the module use open and flexible to as many users as possible 👍

Reply:
Updated in newest commit